START.htm

```diff
@@ -164,35 +164,26 @@ <h2 id="update">Update board with latest software</h2>
     <img src="Docs/images/download-latestimage.png" width="50%" />
 
-    <h3>Step #2: Install compression utility</h3>
-    <p>Download and install <a target="_blank" href="http://www.7-zip.org/download.html">7-zip</a>.</p>
-    <img src="Docs/images/download-7zip.png" width="50%" /><br />
-    <img src="Docs/images/run-7zip-installer.png" width="50%" /><br />
-
-    <h3>Step #3: Decompress the image</h3>
-    <p>Use 7-zip to decompress the SD card .img file</p>
-    <img src="Docs/images/extract-latestimage.png" width="50%" /><br />
-
-    <h3>Step #4: Install SD card programming utility</h3>
-    <p>Download and install <a target="_blank" href="https://launchpad.net/win32-image-writer/+download">Image Writer for Windows</a>. Be sure to download the binary distribution.</p>
-    <img src="Docs/images/download-win32diskimager.png" width="50%" /><br />
-    <img src="Docs/images/extract-win32diskimager.png" width="50%" /><br />
+    <h3>Step #2: Install SD card programming utility</h3>
+    <p>Download and install <a target="_blank" href="https://etcher.io/">Etcher</a>.</p>
+    <img src="Docs/images/download-etcher.png" width="50%" /><br />
+    <img src="Docs/images/install-etcher.png" width="50%" /><br />
     <p>Some general help on programming SD cards can be found on the <a target="_blank" href="http://help.ubuntu.com/community/Installation/FromImgFiles">Ubuntu Image Writer page</a>. 
 
-    <h3>Step #5: Connect SD card to your computer</h3>
+    <h3>Step #3: Connect SD card to your computer</h3>
     <p>Use the provided microSD card to SD adapter or a USB adapter to connect the SD card to your computer.</p>
 
-    <h3>Step #6: Write the image to your SD card</h3>
-    <p>Use either the Ubuntu Image Writer or instructions on its page to write the decompressed image to your SD card.</p>
-    <img src="Docs/images/write-latestimage.png" />
+    <h3>Step #4: Write the image to your SD card</h3>
+    <p>Use Etcher to write image to your SD card.</p>
+    <img src="Docs/images/write-latestimage.png" width="50%" />
 
-    <h3>Step #7: Eject the SD card</h3>
+    <h3>Step #5: Eject the SD card</h3>
     <p>Eject the newly programmed SD card.</p>
 
-    <h3>Step #8: Boot your board off of the SD card</h3>
+    <h3>Step #6: Boot your board off of the SD card</h3>
     <p>Insert SD card into your (powered-down) board, hold down the USER/BOOT button (if using Black) and apply power, either by the USB cable or 5V adapter.</p>
     <p>If using an original white BeagleBone, you are done.</p>
     <p>If using BeagleBone Black and the image is meant to program your on-board eMMC, you'll need to wait while the programming occurs.  When the flashing is complete, all 4 USRx LEDs will be lit solid.  <em>This can take up to 45 minutes.</em>  Power-down your board, remove the SD card and apply power again to be complete.</p>
```

Comment:
Unrelated to the changes you've made here, but I think "apply power again to finish." might sound better than "apply power again to be complete." ?